### PR TITLE
Document that unique property constraints can be composite

### PR DESCRIPTION
**Pull request by Hunterness**
Also updated syntax description for node key constraint to clarify that single property constraints are allowed not just composite.

First commit is cherry-pick of https://github.com/neo4j/neo4j-documentation/pull/1174.

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 1]
```diff
@@ -101,7 +101,7 @@ Tests for the absence of a pattern predicate.
 |
 
 | <<query-plan-create-unique-constraint,CreateUniqueConstraint>>
-| Creates a unique constraint on a property for all nodes having a certain label.
+| Creates a unique constraint on a set of properties for all nodes having a certain label.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc:104]: Also need to update this in QueryPlanTest I think.

File: cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc [hunk 2]
```diff
@@ -56,7 +56,7 @@ However it is worth listing up-front the key areas that are not supported in the
 | Feature | Enterprise | Community
 | <<administration-databases, Multi-database>> | Any number of user databases | Only `system` and one user database
 | Role-based security | <<administration-security, User, Role and Privilege management>> for flexible access control and <<administration-security-subgraph, sub-graph access control>>. | <<administration-security-users, Multi-user management>>. All users have full access rights.
-| Constraints | <<administration-constraints-prop-exist-nodes, Existence constraints>> and <<administration-constraints-node-key, multi-property `NODE KEY` constraints>>.  | Only <<administration-constraints-unique-nodes, single property uniqueness constraints>>
+| Constraints | <<administration-constraints-prop-exist-nodes, Existence constraints>>, <<administration-constraints-unique-nodes, uniqueness constraints>>, and <<administration-constraints-node-key, `NODE KEY` constraints>>.  | Only <<administration-constraints-unique-nodes, uniqueness constraints>>
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc:59]: These links seem broken both here, and in [the real docs](https://neo4j.com/docs/cypher-manual/current/introduction/neo4j-databases-graphs/#:/constraints/examples.adoc/index.adoc#administration-constraints-prop-exist-nodes). Can we fix?

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 3]
```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
...
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON ()-"["r:REL_TYPE"]"-() ASSERT r.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all relationships with a particular type have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring the uniqueness of the combination of node label and property value for a particular property key across all nodes.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: Looks fine, but again these links are broken both here and in the real docs

File: cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc [hunk 4]
```diff
@@ -12,7 +12,8 @@ The following constraint types are available:
 
 *Unique node property constraints*::
 Unique property constraints ensure that property values are unique for all nodes with a specific label.
-Unique constraints do not mean that all nodes have to have a unique value for the properties -- nodes without the property are not subject to this rule.
+For unique property constraints on multiple properties, the combination of the property values is unique.
+Unique constraints do not mean that all nodes have to have a unique value for the properties -- nodes without all properties are not subject to this rule.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc]: Maybe reword slightly
```suggestion
Unique constraints do not require all nodes to have a unique value for the properties listed -- nodes without all properties are not subject to this rule.
```

**Review by mnd999 — CHANGES_REQUESTED** [review]
A couple of broken links I think, probably from the refactoring and need to update the QueryPlanTest

File: cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc:59]: https://neo4j.slack.com/archives/C02JR5E4L/p1625820388358600 I believe that the docs team is working on the links (they broke when the section moved)

**Comment by Hunterness** [comment]
For docs team: 
This adds links down to the administration chapter which might 
a) be affected by the link weirdness
b) need updating when that section of the administration chapter is moved
Not sure if either, bu tjust as a heads-up :)

**Comment by Hunterness** [comment]
For docs team:
This adds links down to the constraints and administration chapters which might
a) be affected by the link weirdness
b) need updating when that section of the administration chapter is moved
Not sure if either, but just as a heads-up :)

File: cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc (hunk 2 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/general/databases-and-graphs.asciidoc:59]: I'm also moving over my comment around links that I added to the 4.3 pr of the first commit

**Review by mnd999 — APPROVED** [review]

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 5]
```diff
@@ -16,6 +16,40 @@ Replacement syntax for deprecated and removed features are also indicated.
...
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
+ON (n:LabelName)
+ASSERT (n.propertyName_1, …, n.propertyName_n) IS UNIQUE
+[OPTIONS "{" option: value[, ...] "}"]
+----
+a|
+Unique property constraints now allow multiple properties, ensuring the combination of property values are unique.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]:
```suggestion
Unique property constraints now allow multiple properties, ensuring that the combination of property values are unique.
```

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc (hunk 1 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc:104]:
```suggestion
| Creates a unique constraint on a set of properties for all nodes with a certain label.
```
I do realize that changing this requires changing in a lot of other places, so I understand if you disregard. But it is not correct as it is.

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 6]
```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
...
 |<<query-create, CREATE>>                    | Writing     |  Create nodes and relationships.
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all nodes with a particular label have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1, ..., n.propN) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]:
```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc (hunk 3 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/glossary.asciidoc]:
```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc [hunk 7]
```diff
@@ -47,12 +48,11 @@ Databases containing one of these constraint types cannot be opened using Neo4j
 
 Creating a constraint has the following implications on indexes:
 
-* Adding a unique property constraint on a property will also add a <<administration-indexes-create-a-single-property-index-for-nodes, single-property index>> on that property, so such an index cannot be added separately.
-* Adding a node key constraint for a set of properties will also add a <<administration-indexes-create-a-composite-index-for-nodes, composite index>> on those properties, so such an index cannot be added separately.
+* Adding a node key or unique property constraint on a single property will also add a <<administration-indexes-create-a-single-property-index-for-nodes, single-property index>> on that property, so such an index cannot be added separately.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc]:
```suggestion
* Adding a node key or unique property constraint on a single property also adds a <<administration-indexes-create-a-single-property-index-for-nodes, single-property index>> on that property and therefore, such an index cannot be added separately.
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc [hunk 8]
```diff
@@ -47,12 +48,11 @@ Databases containing one of these constraint types cannot be opened using Neo4j
...
+* Adding a node key or unique property constraint for a set of properties will also add a <<administration-indexes-create-a-composite-index-for-nodes, composite index>> on those properties, so such an index cannot be added separately.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc]:
```suggestion
* Adding a node key or unique property constraint for a set of properties also adds a <<administration-indexes-create-a-composite-index-for-nodes, composite index>> on those properties and therefore,  such an index cannot be added separately.
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc [hunk 9]
```diff
@@ -47,12 +48,11 @@ Databases containing one of these constraint types cannot be opened using Neo4j
...
 * Cypher will use these indexes for lookups just like other indexes.
   Refer to <<administration-indexes-search-performance>> for more details on indexes.
-* If a unique property constraint is dropped and the single-property index on the property is still required, the index will need to be created explicitly.
-* If a node key constraint is dropped and the composite-property index on the properties is still required, the index will need to be created explicitly.
+* If a node key or unique property constraint is dropped and the backing index is still required, the index will need to be created explicitly.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/index.adoc]:
```suggestion
* If a node key or unique property constraint is dropped and the backing index is still required, the index need to be created explicitly.
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc [hunk 10]
```diff
@@ -86,6 +101,8 @@ The preferred way of dropping a constraint is by the name of the constraint.
 This drop command is optionally idempotent, with the default behavior to throw an error if you attempt to drop the same constraint twice.
 With the `IF EXISTS` flag, no error is thrown and nothing happens should the constraint not exist.
 
+Dropping a constraint requires <<administration-security-administration-database-constraints, the `DROP CONSTRAINT` privilege>>.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]:
```suggestion
Dropping a constraint requires the <<administration-security-administration-database-constraints, `DROP CONSTRAINT`>>  privilege.
```

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc [hunk 11]
```diff
@@ -136,20 +160,24 @@ An old way of dropping a node key constraint was to drop the constraint by speci
...
+ON (n:LabelName)
+ASSERT (n.propertyName_1, … n.propertyName_n) IS NODE KEY
 ----
 
 
 [[administration-constraints-syntax-list]]
 == Syntax for listing constraints
 
 List constraints in the database, either all or filtered on constraint type.
+This requires <<administration-security-administration-database-constraints, the `SHOW CONSTRAINT` privilege>>.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]:
```suggestion
This requires  the<<administration-security-administration-database-constraints, `SHOW CONSTRAINT` >> privilege.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 12]
```diff
@@ -93,11 +93,11 @@ This constraint will create an accompanying index.
 //
 
 CREATE CONSTRAINT uniqueness ON (p:Person)
-       ASSERT p.age IS UNIQUE
+       ASSERT (p.firstname, p.age) IS UNIQUE
 ###
 
-Create a unique property constraint on the label `Person` and property `age` with the name `uniqueness`.
-If any other node with that label is updated or created with a `age` that
+Create a unique property constraint on the label `Person` and properties `firstname` and `age` with the name `uniqueness`.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: Is the name `uniqueness` the name of the property? In that case, I'd say:
```suggestion
Create a unique property constraint with the name `uniqueness` on the label `Person` and properties `firstname` and `age`.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 13]
```diff
@@ -93,11 +93,11 @@ This constraint will create an accompanying index.
...
+If any other node with that label is updated or created with a `firstname` and `age` combination that
 already exists, the write operation will fail.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
already exists, the write operation fails.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 14]
```diff
@@ -93,11 +93,11 @@ This constraint will create an accompanying index.
...
 This constraint will create an accompanying index.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
This constraint creates an accompanying index.
```
This should really be updated everywhere, but I understand if that is out of scope for the PR.

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 15]
```diff
@@ -173,21 +173,23 @@ CREATE CONSTRAINT ON (p:Person)
 (★) Create a node key constraint on the label `Person` and properties `firstname` and `surname`.
 If a node with that label is created without both `firstname` and `surname`
 or if the combination of the two is not unique,
-or if the `firstname` and/or `surname` labels on an existing node with the `Person` label
+or if the `firstname` and/or `surname` properties on an existing node with the `Person` label
 is modified to violate these constraints, the write operation will fail.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
is modified to violate these constraints, the write operation fails.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 16]
```diff
@@ -173,21 +173,23 @@ CREATE CONSTRAINT ON (p:Person)
...
+This constraint will create an accompanying index.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
This constraint creates an accompanying index.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 17]
```diff
@@ -173,21 +173,23 @@ CREATE CONSTRAINT ON (p:Person)
...
 
-(★) Create a node key constraint on the label `Person` and properties `name` and `surname` with the name `node_key`.
-If a node with that label is created without both `name` and `surname`
-or if the combination of the two is not unique,
-or if the `name` and/or `surname` labels on an existing node with the `Person` label
+(★) Create a node key constraint on the label `Person` and property `firstname` with the name `node_key`.
+If a node with that label is created without the `firstname` property
+or if the value is not unique,
+or if the `firstname` property on an existing node with the `Person` label
 is modified to violate these constraints, the write operation will fail.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
is modified to violate these constraints, the write operation fails.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 18]
```diff
@@ -173,21 +173,23 @@ CREATE CONSTRAINT ON (p:Person)
...
+This constraint will create an accompanying index.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
This constraint creates an accompanying index.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 19]
```diff
@@ -208,6 +210,7 @@ DROP CONSTRAINT uniqueness
 ###
 
 Drop the constraint with the name `uniqueness`, throws an error if the constraint does not exist.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
Dropping the constraint with the name `uniqueness`, throws an error if the constraint does not exist.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 20]
```diff
@@ -208,6 +210,7 @@ DROP CONSTRAINT uniqueness
...
+If the constraint has an accompanying index, that will also be dropped.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
If the constraint has an accompanying index, that is also dropped.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 21]
```diff
@@ -216,5 +219,6 @@ DROP CONSTRAINT uniqueness IF EXISTS
 ###
 
 Drop the constraint with the name `uniqueness` if it exists, does nothing if it does not exist.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
Dropping the constraint with the name `uniqueness` if it exists, does nothing if it does not exist.
```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala [hunk 22]
```diff
@@ -216,5 +219,6 @@ DROP CONSTRAINT uniqueness IF EXISTS
...
+If the constraint has an accompanying index, that will also be dropped.
```
Comment (AlexicaWright) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]:
```suggestion
If the constraint has an accompanying index, that is also dropped.
```

**Review by AlexicaWright — COMMENTED** [review]
Some comments, please disregard if changing means too much work in other places.

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc (hunk 10 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]: is this gonna be visible as a link? since it is also formatted with `?

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala (hunk 12 above)
Comment (Hunterness) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: it is the name of the index

Comment (Hunterness) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: I think almost all of these have the index name last

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala (hunk 14 above)
Comment (Hunterness) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: I'll update those you have marked at least (and then see if there are any left in this file)

File: cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc (hunk 10 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc]: I think so, it works in manual-modeling.

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala (hunk 12 above)
Comment (AlexicaWright) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: I meant the name of the constraint, sorry! If this is a consistent structure, then keep it, even if it's a bit misleading.

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 23]
```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
...
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON ()-"["r:REL_TYPE"]"-() ASSERT r.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all relationships with a particular type have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: Ah, my butterfingers! One too many _that_. 
...Create a constraint that ensures the uniqueness...

**Review by AlexicaWright — COMMENTED** [review]
My apologies for fixing my own fix.

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala (hunk 12 above)
Comment (Hunterness) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala]: I think it is only on the ref card so I think we could update it for both index and constraints

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 24]
```diff
@@ -25,9 +25,9 @@ This section comprises a glossary of all the keywords -- grouped by category and
...
 |<<query-create, CREATE>>                    | Writing     |  Create nodes and relationships.
 |<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all nodes with a particular label have a certain property.
-|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
+|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: This sentence could keep the extra _that_ : "...that ensures that all nodes.."

**Review by AlexicaWright — COMMENTED** [review]
One little _that_ got lost. Otherwise it looks really good!

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc (hunk 24 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/glossary.asciidoc]:
```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```
